packages/base/src/3dview/mainview.tsx

```diff
@@ -1340,3 +1362,14 @@ export class MainView extends React.Component<IProps, IStates> {
   private _contextMenu: ContextMenu;
   private _loadingTimeout: ReturnType<typeof setTimeout> | null;
 }
+
+export function toggleWireframe(this: MainView) {
```

Comment:
Why is function is defined outside of the class and not its method?

Reply:
I was finding it difficult to access this method outside this file, hence did this for the time being.
I'd be on my way to define it inside the class and then try to use it across. I believe I'd have questions which I would be asking definitely while I'm on it specifically. Thanks for reviewing this.

Reply:
Marking this as resolved as now it is being handled inside `_onViewChanged`